server/forms.py

```diff
@@ -0,0 +1,58 @@
+from django import forms
+from .models import User, Player, Membership, Vaccination
+
+GENDER_CHOICES = [('male', 'Male'), ('female', 'Female'), ('other', 'Other')]
```

Comment:
Should we make the field on the model an `enum` field with these as the choices?

Reply:
done, please check `models.py`